src/main/scripts/src/app/shared/perspectives/documents/post-it/attribute/attribute-property-selection.ts

```diff
@@ -18,12 +18,12 @@
  * -----------------------------------------------------------------------/
  */
 
-export interface AttributePropertyInput {
+export interface AttributePropertySelection {
 
-  id: string;
-  propertyName: string;
-  inputTableX: number;
-  inputTableY: number;
-  element: HTMLInputElement;
+  direction: string;
```

Comment:
What about using `enum` instead? You can even make use of [TypeScript 2.4 latest features](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-2-4.html).